### PR TITLE
added model selection per CV Folds

### PR DESCRIPTION
**Pull request by lauravarsandan**
added model selection per CV Folds

### CODE REVIEW

File: naive_bayes_model_selection.asv [hunk 1]
```diff
@@ -25,83 +25,220 @@ heatmap(CM);
...
-BestDist=Mdl.ModelParameters.DistributionNames;
-BestWidth=Mdl.ModelParameters.Width;
-BestKernelType=Mdl.ModelParameters.Kernel;
+for f = 5:10
 
+        X=nb_ms(:,1:6);
+        Y=nb_ms(:,7);
+        prior = [0.6999 0.2227 0.0398 0.0376];
+        class_names={'unacc','acc','good','vgood'};
+        rng(1);
```
Comment (teopopescu) [inline naive_bayes_model_selection.asv:42]: great job !

File: naive_bayes_model_selection.asv [hunk 2]
```diff
@@ -25,83 +25,220 @@ heatmap(CM);
...
-c = cvpartition(nb_ms.acceptability,'KFold',num_folds);
+        Mdl = fitcnb(X,Y,...
+            'ClassNames',class_names,...
+            'Prior',prior,...
+            'OptimizeHyperparameters','all',...
+            'HyperparameterOptimizationOptions',struct('AcquisitionFunctionName','expected-improvement-plus',...
+            'Kfold',f));
+
+        % Get the best hyper parameters
+        BestDist=Mdl.ModelParameters.DistributionNames;
```
Comment (teopopescu) [inline naive_bayes_model_selection.asv:52]: what is this?

File: naive_bayes_model_selection.asv (hunk 1 above)
Comment (lauravarsandan) [reply naive_bayes_model_selection.asv:42]: bla

File: naive_bayes_model_selection.asv (hunk 2 above)
Comment (lauravarsandan) [reply naive_bayes_model_selection.asv:52]: bla

**Pull request merged**